Assets/MixedRealityToolkit.Tests/PlayModeTests/RenderingTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.Rendering;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class RenderingTests : BasePlayModeTests
+    {
+        private const float VisualizatonWaitTime = 1.0f;
+        private readonly Vector3 LightDirection = new Vector3(-0.5f, -0.5f, 0.0f);
+
+        /// <summary>
+        /// Tests if <see cref="MaterialInstance"/> can be instantiated and manipulated at runtime.
+        /// </summary>
+        [UnityTest]
+        public IEnumerator TestMaterialInstance()
+        {
+            var light = InstantiateDirectionalLight(LightDirection);
```

Comment:
out of curiosity is a light needed?

Reply:
No it isn't, but I added it to inspect what was going on in the editor a bit better. I can remove it.

Reply:
I figured, feel free to keep it. But add comment that it's just for developer visually inspecting test or something